src/pybind/mgr/dashboard/frontend/src/app/shared/components/form-modal/form-modal.component.ts

```diff
@@ -7,11 +7,17 @@ import { BsModalRef } from 'ngx-bootstrap/modal';
 import { CdFormBuilder } from '../../forms/cd-form-builder';
 
 interface CdFormFieldConfig {
-  type: 'textInput';
+  type: 'textInput' | 'select';
   name: string;
```

Comment:
Found a little issue (not caused by this PR).
The first `ngSwitchCase` in the template is `inputText`, but here we have `textInput`.
The only consumer so far is correct though: 
https://github.com/ceph/ceph/blob/6bab7c8002ba1d07287046dc39861ffbf7621b73/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-directories/cephfs-directories.component.ts#L280-L287

Reply:
Good spotting, have fixed that issue.